### PR TITLE
Case study as single page entry

### PR DESCRIPTION
**Pull request by IshaDakota**
#398 
simpler version of case study.

### CODE REVIEW

**Comment by amitaibu** [comment]
Yeah, I like this approach better :)

**Comment by amitaibu** [comment]
@IshaDakota would you be able to finalize this PR?

**Comment by IshaDakota** [comment]
fireshot capture 9 - one database to rule them all - http___0 0 0 0_9000_case-studies_openscholar_

**Comment by IshaDakota** [comment]
fireshot capture 10 - gambling on cow insurance - http___0 0 0 0_9000_case-studies_ibli_ copy

**Comment by IshaDakota** [comment]
I took some liberties with design here not because I think it's that good, but because I wanted to visually suggest that if these will be showcased, the should have a degree of visual sophistication that reflects the quality of our work. Maybe a designer could look closely and suggest some refinement.

@kazabooboo @RachelBaram for our call tomorrow - take a look.

**Comment by IshaDakota** [comment]
Things to consider:
-  A landing page for case studies... how are they tied into the menu?
-  tags... the way they are now isn't really functional - maybe a "see also"?
-  didn't check these on mobile - there's something funky with font-size on lists that may need to be fixed site wide.
-  In general I don't like the vertical rhythm (for the blogs as well) font-size + line-height + p top-margin don't create enough separation between paragraphs and text seems to spread out.

File: app/_scss/pages/_case-page.scss [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+  h2,
+  h3 {
+    @extend .headers-styling;
+  }
+
+  h1 {
+    @extend .h1-styling;
+    border-bottom: 1px solid;
+
+    //margin-bottom: 6px;
```
Comment (amitaibu) [inline app/_scss/pages/_case-page.scss]: remove if unused

**Comment by amitaibu** [comment]
@IshaDakota @RachelBaram After some consideration and talking with Brice, I think we should finish this one before we re-design stuff.

It seems it's very close the being merge - which will probably be the quickest option. After we'll have it in, we can iterate over it, and polish.

**Comment by IshaDakota** [comment]
@amitaibu 
I'm fine with this option and I discussed with Rachel today and Adar last week - here are my thoughts about process (for comment):

1. Let's have someone with a "design eye" look at the current draft to give us critique (either in sketch or in writing)  of the current draft. The person should pay particular attention to the visual elements that will call attention to the main points of the studies (e.g. pull quotes or other types of emphasis). We understand that we are working within the current look and feel of the website and not designing something new. I have no opinion on who that person should be (or if it's internal or external) - but should be someone with a keen eye.

2. We will need a landing page for case studies and insert that page into the menu. - I am willing to provide a simple draft of that within this PR - or let someone else do it as they are available. 

**The below items refer to both case studies and blogs:**
3. Typography - spacing and vertical rhythm is way off on these pages - making the longest text hard to read. After step 1 is completed, I am happy to apply some sort of standard rule to the typography so we see how it looks.

4. Image handling/placement/captions - again, in part a vertical rhythm issue, but also awkward handling of images make the graphics not speak as well as they could. This could use some design love.

If we handle these issues I think that we will have presentable case studies **that can be easily turned into PDFs.** If this represents the list of needs, Let's make each point an issue, and I believe that we could handle this week.

**Comment by amitaibu** [comment]
I've optimized the images, and think we need to tackle the points one by one (a.k.a smaller issues), so going to merge and deploy this.

**Pull request merged**